nautobot/extras/tests/git_helper.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import logging
+import os
+import os.path
+import shutil
+import tempfile
+
+from git import Repo
+
+logger = logging.getLogger(__name__)
+
+
+SOURCE_DIR = os.path.join(os.path.dirname(__file__), "git_data")
+
+
+def create_and_populate_git_repository(target_path):
+    """Main entry point to this script."""
```

Comment:
A short summary of what this is doing would be good here for when someone is trying to troubleshoot failing tests